app/subscriber/src/features/press-gallery/PressGallery.tsx

```diff
@@ -1,17 +1,87 @@
 import { DateFilter } from 'components/date-filter';
 import { FolderSubMenu } from 'components/folder-sub-menu';
+import { IContentListAdvancedFilter } from 'features/content/list-view/interfaces';
 import { determineColumns } from 'features/home/constants';
+import moment from 'moment';
 import React from 'react';
 import { useNavigate } from 'react-router-dom';
-import { FlexboxTable, IContentModel, ITableInternalRow, Row } from 'tno-core';
+import { useContent } from 'store/hooks';
+import { useContributors } from 'store/hooks/subscriber/useContributors';
+import { FlexboxTable, IContentModel, IContributorModel, ITableInternalRow, Row } from 'tno-core';
 
 import * as styled from './styled';
 
-/** TODO: Didn't realize I had two different stories - this page will be done in the next ticket*/
 export const PressGallery: React.FC = () => {
   const navigate = useNavigate();
+  const [{ filterAdvanced }, { findContentWithElasticsearch }] = useContent();
+  const [, api] = useContributors();
+  const [pressContributors, setPressContributors] = React.useState<IContributorModel[]>([]);
+  const [results, setResults] = React.useState<any>([]);
   const [selected, setSelected] = React.useState<IContentModel[]>([]);
 
+  const query = (queryText: string, filterAdvanced: IContentListAdvancedFilter) => {
```

Comment:
We have shared methods in tno-core that we should use so that we have a single place to maintain.